torch/_torch_docs.py

```diff
@@ -7327,6 +7327,53 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.vandermonde_matrix,
+           """
+vandermonde_matrix(x, n, increasing=True, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Generate a Vandermonde matrix.
+
+The columns of the output matrix are powers of the input vector. The order of the powers is 
+determined by the increasing boolean argument. Specifically, when increasing is False, the 
+i-th output column is the input vector raised element-wise to the power of :math:`N - i - 1`. Such a 
+matrix with a geometric progression in each row is named for Alexandre-Theophile Vandermonde.
```

Comment:
Nice historical reference. Not "mathematician Alex..."?

Reply:
just pulled it from the numpy docs. IDK how close we want to follow it.